slack_sdk/web/async_client.py

```diff
@@ -621,7 +621,7 @@ async def admin_users_session_getSettings(
             kwargs.update({"user_ids": ",".join(user_ids)})
```

Comment:
This code currently works but the server-side may stop accepting this format in the future. That's the reason why we want to make the changes in this PR.